app/src/main/java/co/tinode/tindroid/db/SqlStore.java

```diff
@@ -127,7 +127,6 @@ public boolean topicDelete(Topic topic, boolean hard) {
                 db.beginTransaction();
 
                 if (hard) {
-                    EditHistoryDb.deleteAll(db, st.id);
```

Comment:
Did EditHistoryDb get released already?
If so, we should perhaps keep this statement.

Reply:
I bumped DB version, the `EditHistoryDb` will be removed on ugrade.